config/capz/kustomization.yaml

```
@@ -51,8 +52,7 @@ replacements:
       index: 1
     select:
       annotationSelector: cert-manager.io/inject-ca-from
-
-- source: # SERVICE_NAMESPACE
+- source:
```

```suggestion
- source: # SERVICE_NAMESPACE
```
Some more comments to preserve.

---

added via 7902bf3d8168dafa5054c010f64c089df370b3ce